includes/admin/views/omise-page-settings.php

```diff
@@ -1,6 +1,24 @@
 <div class="wrap omise">
+	<style>
+		.omise-notice-testmode {
+			background: #ffce00;
+			color: #575D66;
+			border: 1px solid #efc200;
+			border-left-width: 4px;
+		}
+	</style>
 	<h1><?php echo $title; ?></h1>
+
+	<?php $page->display_messages(); ?>
+
+	<?php if ( 'yes' === $settings['sandbox'] && $settings['account_email'] ) : ?>
+		<div class="notice omise-notice-testmode">
+			<p><?php echo _e( 'You are in test mode. No actual payment is made in this mode', 'omise' ); ?></p>
```

Comment:
Is there a corresponding similar warning at the checkout?

Reply:
For the test-mode warning bar, at the moment, no (maybe another PR)